app/assets/javascripts/discourse/tests/fixtures/theme-setting-schema-data.js

```diff
@@ -179,10 +179,6 @@ export default function schemaAndData(version = 1) {
         boolean_field: {
           type: "boolean",
         },
-        enum_field: {
```

Comment:
Hmm if you removed enum fields in this version, which version should have it?

Reply:
I moving away from this “fixtures” in another file pattern and instead prefer to declare the data within the test itself.